libtelemetry/src/main/java/com/mapbox/android/telemetry/MapLoadEvent.java

```diff
@@ -88,6 +100,8 @@ private MapLoadEvent(Parcel in) {
     cellularNetworkType = in.readString();
     byte wifiVal = in.readByte();
     wifi = wifiVal == 0x02 ? null : wifiVal != 0x00;
+    sdkIdentifier = in.readString();
```

Comment:
would it be fair to assume that readString() on this line returns null if this field is not available?